### PR TITLE
Update Device Memory API docs

### PR DESCRIPTION
**Pull request by sirovenmitts**
I've fixed some spelling errors, and tried to reformat the content to sound conversational. I've also capitalized RAM everywhere it is used.

### CODE REVIEW

**Comment by ultrafez** [comment]
In my opinion it'd be helpful to reference that Client Hints refers to HTTP headers the first time they are mentioned - if the reader is not familiar with them then it helps give them a rough idea of the concept without having to follow the link to read in detail.

File: files/en-us/web/api/device_memory_api/index.html [hunk 1]
```diff
@@ -6,25 +6,21 @@
...
-</pre>
+<pre class="brush:js notranslate">var RAM = navigator.deviceMemory;</pre>
 
-<p>Both of these will return the same result.</p>
+<h3 id="Client_Hints">Client Hints</h3>
 
-<h3 id="Client_Hints_Header">Client Hints Header</h3>
-
-<p>You also may use <a href="/en-US/docs/Glossary/Client_hints">Client Hints</a> directive <code>Device-Memory</code> to retrieve the same approximate RAM capacity.</p>
+<p>You may also use the <a href="/en-US/docs/Glossary/Client_hints">Client Hints</a> <code>Device-Memory</code> directive to retrieve the same approximate RAM capacity.</p>
```
Comment (hamishwillee) [inline files/en-us/web/api/device_memory_api/index.html:23]: All good, except I think it is important to clarify that we mean Client Hints HTTP Header. So perhaps best way would be 

```
<p>You may also use the <a href="/en-US/docs/Glossary/Client_hints">Client Hints</a> HTTP header with the <code>Device-Memory</code> directive to retrieve the same approximate RAM capacity.</p>

``

Comment (sirovenmitts) [reply files/en-us/web/api/device_memory_api/index.html]: Thanks for the feedback! I've added the suggested language to the page.

**Review by sideshowbarker — APPROVED** [review]
Excellent.

**Comment by sideshowbarker** [comment]
Thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Pull request merged**